### DISCUSSION TITLE
Composition with React + React Native

### DISCUSSION THREAD

**Discussion by yuriazevedo11** [storybookjs/storybook#15792] [General] [answered]
Hi :)

I was wondering if composition supports a React.js Storybook composed with a React Native Storybook.

Ex.: I'm running the web version on `http://localhost:6006` and the native one on `http://localhost:7007`. I tried to add this to my storybook/main.js on the web storybook:
```js
refs: {
  react: {
    title: 'Native Env',
    url: 'http://localhost:7007',
  },
},
```
But I got no success. Is this possible? If not, is there any plans to add support to this?

**Comment by dannyhw** [comment, marked as answer]
its not possible to render native mobile components on the web you could only do something like this with a react native web storybook and a react storybook (already should be possible).

**Comment by jonniebigodes** [comment]
Thanks for following up on this @dannyhw ! Appreciate it.
